extensions/amp-story/1.0/bookend/components/bookend-component-interface.js

```diff
@@ -34,10 +34,10 @@ export class BookendComponentInterface {
   build(opt_componentJson) {}
 
   /**
-   * Builds the template for the component.
+   * Builds the HTML element for the component.
    * @param {../bookend-component.BookendComponentDef} opt_componentJson
    * @param {!Document} opt_doc
```

Comment:
``!Document`` and ``opt_doc`` have two different meanings. Is this required or optional?
If optional, please remove the ``opt_`` prefix and use default parameters instead.
Should we fix this in this PR?

Reply:
As we are not using the variable in the method signature of the interface, we get a linter error for not using the parameter, so we must prepend it with opt_ 😕. Maybe there is another approach we could take? @newmuis might know about this

Reply:
This should be fixed by adding the `Interface` suffix to the interface name. Big footgun in our type rules.

Reply:
Sorry not sure I understood. Did you mean adding it to the type rules?